.rubocop.yml

```diff
@@ -195,9 +195,9 @@ Layout/IndentationConsistency:
 #
 #   # do something
 # end
-Layout/MultilineMethodCallIndentation:
-  Enabled: true
-  EnforcedStyle: indented
+# Layout/MultilineMethodCallIndentation:
+#   Enabled: true
+#   EnforcedStyle: indented_relative_to_receiver
```

Comment:
I didn't understand why this rule was removed. Can you point out what issues this rule was causing?

This is a good rule in my opinion.

The following is bad/unformatted code in my opinion:

```ruby
    def send_devise_notification(notification, *args)
      devise_mailer
      .send(notification, self, *args)
      .deliver_later(queue: "devise_email")
    end
```

This rule will correct and format it and make it more readable like so:
```ruby
    def send_devise_notification(notification, *args)
      devise_mailer
        .send(notification, self, *args)
        .deliver_later(queue: "devise_email")
    end
```

I suggest we keep this rule as it was.

Reply:
With the earlier code

**good**
```ruby
user_name = User.
  where(type: :bot).
  first.
  name
```

**bad**
```ruby
user_name = User
              .where(type: :bot)
              .first
              .name
```

IMO the second one is more readable. That's why added `indented_relative_to_receiver`.

Reply:
If rubocop can autofix it then I'll go with whatever rubocop can do without caring about what the style it. That's because with rubocop atleast we have some consistency. Without rubocop some folks will do this way and some folks will do that way.